src/app/components/TodoList.js

```diff
@@ -0,0 +1,119 @@
+import TodoApi from '../services/TodoApi';
+import SelectColor from './SelectColor';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+export default class TodoList extends React.Component {
+  constructor(props) {
```

Comment:
using Injection is preferable over creating a class object
```suggestion
constructor(
  private todoApi: TodoApi,
  props
) {
```

Reply:
then you can refer to this like this.todoApi anywhere in this component

Reply:
thats more an Angular practice. I would move this to a state object